src/Sarif.Viewer.VisualStudio/Models/CallTreeNode.cs

```diff
@@ -1,14 +1,54 @@
 ﻿// Copyright (c) Microsoft. All rights reserved. 
 // Licensed under the MIT license. See LICENSE file in the project root for full license information. 
 
+using System;
 using System.Collections.Generic;
+using System.IO;
 using Microsoft.CodeAnalysis.Sarif;
 
 namespace Microsoft.Sarif.Viewer.Models
 {
-    public class CallTreeNode
+    public class CallTreeNode : CodeLocationObject
     {
-        public AnnotatedCodeLocation Location { get; set; }
+        private AnnotatedCodeLocation _location;
+
+        public AnnotatedCodeLocation Location
+        {
+            get
+            {
+                return this._location;
```

Comment:
We probably shouldn't use "this."s to be consistent. I'll remove them in the files in this PR. Likely there are other places in the sln where I've introduced "this."'s. I guess we can remove them as we touch the files.